Tests/js/apps/assets/ez-platformuiapp-tests.js

```diff
@@ -1459,15 +1500,12 @@ YUI.add('ez-platformuiapp-tests', function (Y) {
             delete this.capiMock;
         },
 
-        "Should reset the user and provide the response when logging out": function () {
-            var logOutResponse = {},
-                logOutError = {};
-
+        _mockExpectations: function () {
```

Comment:
that's a bad name because your method does too many things, you should have one to configure the `capiMock` and another one for the user mock.